docs/guide/images.md

```diff
@@ -227,8 +227,7 @@ a custom extension that lets you display and download files in your templates.
 First, enable adding SVG files to content by removing them from the blacklist 
 of allowed MIME types.
 
-To do it, comment out the relevant line under `ibexa.site_access.config.default.io.file_storage.file_type_blacklist`
-in `Core/Resources/config/default_settings.yml`.
+To do it, overwrite `ibexa.site_access.config.default.io.file_storage.file_type_blacklist` defined in `Core/Resources/config/default_settings.yml` so that `svg` is removed from the Blacklist. This can be done per Site Access using [SiteAccess-aware configuration](multisite/siteaccess_aware_configuration.md).
```

Comment:
```suggestion
To do it, overwrite `ibexa.site_access.config.default.io.file_storage.file_type_blacklist` defined in `Core/Resources/config/default_settings.yml` so that `svg` is removed from the blacklist.
You can do it per Site Access by using [SiteAccess-aware configuration](multisite/siteaccess_aware_configuration.md).
```